c2corg_ui/static/js/documentdetailsview.js

```diff
@@ -72,7 +70,16 @@ app.ViewDetailsController = function($uibModal, $compile, $scope) {
    */
   this.scope_ = $scope;
 
-}
+  this.scope_['document'] = {
+    'orientations' : this['orientations'],
```

Comment:
What is ``orientations``?
A route-specific attribute? If yes, can't we avoid it since the directive is not route-specific and should be generic.